release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -180,6 +180,19 @@ For more information, see xref:../post_installation_configuration/cluster-capabi
 [id="ocp-4-11-web-console"]
 === Web console
 
+[id="ocp-4-111-developer-perspective"]
+==== Developer Perspective
+
+* With this update, in the developer perspective, you can add your GitHub repository containing pipelines to the Openshift Container Platform cluster. This allows you to run pipelines and tasks from your GitHub repository on the cluster when relevant Git events, such as push or pull requests are triggered.
+
+** In the administrator perspective, you can configure your GitHub application with the OpenShift cluster to use a pipeline as code. This configuration allows you to execute a set of tasks required for build deployment.
+
+* With this update, you can create a customized pipeline using your own set of curated tasks. You can search, install, and upgrade your tasks directly from the developer console.
+
+* With this update, in the web terminal you can now have multiple tabs, view bash history, and the web terminal will remain open until you close the browser window or tab.
+
+* With this update, in the  *Add+* page of the developer perspective a new menu added to share project and Helm Chart repositories that allows to add or remove users to the project.
```

Comment:
- I think there are two spaces before the *Add+*
- add comma after "perspective."